consensus/istanbul/announce/valproxy.go

```diff
@@ -20,43 +19,6 @@ type ValProxyAssigmnentProvider interface {
 	GetValProxyAssignments(valAddresses []common.Address) (map[common.Address]*enode.Node, error)
 }
 
-type proxyVPAP struct {
```

Comment:
shouldn't we remove the whole concept of "ValProxyAssigmnentProvider". I understand is to "assign one of my proxies to another validator", basically tell them which is my "enode address" but it would actually be one of my proxy's address.

Without proxies, you just send them your address